include/dmlc/base.h

```diff
@@ -30,18 +30,19 @@
 #endif
 
 /*!
- * \brief Whether to use customized logger,
+ * \brief whether to use customized logger,
  * whose output can be decided by other libraries.
  */
 #ifndef DMLC_LOG_CUSTOMIZE
 #define DMLC_LOG_CUSTOMIZE 0
 #endif
 
 /*!
- * \brief Wheter to print stack trace for fatal error,
+ * \brief wheter to print stack trace for fatal error,
```

Comment:
Thanks. Swapping the case is just to align with other comments, e.g. line 25, 48, 53, etc.